modules/react/tabs/lib/TabsItem.tsx

```diff
@@ -0,0 +1,217 @@
+import * as React from 'react';
+
+import {colors, space, type, borderRadius, iconColors} from '@workday/canvas-kit-react/tokens';
+import {
+  createComponent,
+  focusRing,
+  hideMouseFocus,
+  styled,
+  StyledType,
+  useModelContext,
+  createHook,
+  composeHooks,
+  ExtractProps,
+  ellipsisStyles,
+  EllipsisText,
+} from '@workday/canvas-kit-react/common';
+import {Box} from '@workday/canvas-kit-labs-react/common';
+import {OverflowTooltip} from '@workday/canvas-kit-react/tooltip';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {useListRegisterItem} from './list';
+import {useRovingFocus} from './cursor';
+import {isSelected, useSelectionItem} from './selection';
+import {useOverflowMeasureItem} from './overflow';
+
+import {TabsModelContext} from './Tabs';
+import {TabsModel} from './useTabsModel';
+export interface TabsItemProps extends ExtractProps<typeof Box, never> {
+  /**
+   * Optionally pass index to tab item. This should be done if `Tabs.Item` components were created
+   * via a `Array::map` function. This index will ensure keyboard navigation works even if items are
+   * inserted out of order.
+   */
+  index?: number;
+  /**
+   * The label text of the Tab.
```

Comment:
if it's children can it be anything? not just label text?

Reply:
It can be anything. It is the contents of the tab. I can update to be more clear